src/components/Board.js

```diff
@@ -5,16 +5,14 @@ import PropTypes from 'prop-types';
 
 
 const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
-  // squares is a 2D Array, but 
-  //  you need to return a 1D array
-  //  of square components
+  const squareArray = squares.flat();
+  const squareComponents = squareArray.map(s => <Square id={s.id} value={s.value} onClickCallback={onClickCallback} />)
```

Comment:
Don't forget to add a key prop. You will get a warning in the console about each square having a unique key. `key={s.id}`